api_eucovidcert.yaml

```diff
@@ -0,0 +1,65 @@
+swagger: "2.0"
+info:
+  version: 0.0.1
+  title: EU Covid Certificate service for IO App
+basePath: /eucovidcert/api/v1
+schemes:
+  - https
+security:
+  - Bearer: []
+paths:
+  /certificate:
+    post:
+      operationId: getCertificate
+      summary: Retrieve a certificate for a given Citizen
+      description: |-
+        Given a Citizen's fiscal code and an OTP (the auth code previously sent via IO Message), a Certificate is returned with ID, encoded QR-code, expiration date and a markdown text with all meaningful information to be shown to Citizens.
+      parameters:
+        - name: accessData
+          in: body
+          schema:
+            $ref: "#/definitions/GetCertificateParams"
+      responses:
+        '200':
+          description: |-
+            A Certificate exists and it's found for the given access data. It is retrieved regardless of it's expired or its current status
+          schema:
+            $ref: "#/definitions/Certificate"
+        '400':
+          description: Payload has bad format
+        '403':
```

Comment:
Add `401`, this is an authenticated endpoint and may fail

Reply:
Done in [6c1dbbd](https://github.com/pagopa/io-backend/pull/803/commits/6c1dbbd598b9861837621ef0917f33e4d199748a)